build/dependencies.props

```diff
@@ -1,5 +1,6 @@
 ﻿<Project>
   <PropertyGroup>
+    <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
     <CommandLineUtilsVersion>1.2.0-*</CommandLineUtilsVersion>
     <CoreFxVersion>4.3.0</CoreFxVersion>
```

Comment:
The only package using this is `System.Runtime.Serialization.Primitives` and its latest version on NuGet.org is 4.3.0, it doesn't appear on our feeds. Maybe this property just needs a more accurate name?